Reporting/M5 Document/StepByStep.md

```diff
@@ -0,0 +1,97 @@
+ -   Getting started:
+
+        -   If this is the first time you are pulling from the repo,
+            > open a new command line and run npm install (make sure you
+            > install Node.js and the npm package manager first if you
+            > don\'t have this on your system).
+
+        -   To start up the server, run npm run dev.
```

Comment:
Change to `npm run accord`. Everyone needs to have access to the API keys in the repo or nothing will work. These include the keys for `Ably`, `Clerk` and `ngrok`